packages/mobile/e2e/src/RedeemInviteAndVerify.spec.js

```diff
@@ -0,0 +1,66 @@
+import dismissBanners from './utils/banners'
+import { enterPinUi, enterPinUiIfNecessary, sleep, waitForElementId } from './utils/utils'
+import { receiveSms } from './utils/twilio'
+
+const VERIFICATION_COUNTRY = 'US'
+const VERIFICATION_PHONE_NUMBER = '4698442277'
+const EXAMPLE_NAME = 'Test Name'
+
+const RUN_VERIFICATION_TEST = !!process.env.INVITE_CODE
+
+describe('Redeem invite code and verify number', () => {
+  beforeEach(dismissBanners)
+
+  it('Redeem invite code and verify number', async () => {
+    if (!RUN_VERIFICATION_TEST) {
+      return
+    }
+    for (let i = 0; i < 3; i++) {
+      await element(by.id('Education/progressButton')).tap()
+    }
+
+    await element(by.id('CreateAccountButton')).tap()
+
+    // Accept Terms
```

Comment:
I wonder if we can re-use https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/e2e/src/usecases/NewAccountOnboarding.js here?